introduction.md

```diff
@@ -0,0 +1 @@
+Hello World. I'm Jesus, 24 y-old. Just another web developer learning at CodeSchool.com
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.